bruv.py

```diff
@@ -57,9 +63,35 @@ def ioctl_GWINSZ(fd):
 port = conf.get("port", 29418)
 query = conf.get("query", "")
 template_path = str(conf.get("template_file", "display.tmpl"))
+db_path = str(conf.get("db_file", "bruv.db"))
 
 PATCH_SET_INFO_RE = re.compile(r"^(?:Patch Set|Uploaded patch set) ([\d]+)")
 
+class DbAPI(object):
```

Comment:
The class should get the db_path in the constructor.
For convenience add get_data_store() that sends the proper variable.